doc/manual/variables-and-scoping.rst

```diff
@@ -305,15 +359,15 @@ macro definition need not come before its inner usage:
 
 .. doctest::
 
-    julia> f = y -> x + y
-    (::#1) (generic function with 1 method)
+    julia> f = y -> y + a
+    (::#2) (generic function with 1 method)
```

Comment:
> `julia> f = y -> y + a`
> `(::#1) (generic function with 1 method)`

is also what I get in a fresh REPL, but using that as the expected output causes the doctest to fail for me (I'm seeing something similar at line 367.) Changing the expected output as I've done causes the doctests to pass.